src/DocumentDbTests/Bugs/Bug_1875_duplicated_array_field_test.cs

```diff
@@ -0,0 +1,40 @@
+using System.Collections.Generic;
+using System.Linq;
+using Marten.Services;
+using Marten.Testing.Documents;
+using Marten.Testing.Harness;
+using NpgsqlTypes;
+using Shouldly;
+using Weasel.Core;
+using Xunit;
+
+namespace DocumentDbTests.Bugs
+{
+    public class Bug_1875_duplicated_array_field_test : BugIntegrationContext
+    {
+        [Fact]
+        public void query_on_duplicated_array_field_test()
```

Comment:
**SUGGESTION:** @mysticmind, it'd be worth also checking the `Guid` array as in the original array. We had cases in the past that types like string, guids, dates were failing, while for numbers, it was fine, and we were lacking tests for them.

Reply:
@oskardudycz, the issue was reproducible for the unit test which I have added so we are good.

Reply:
There is no harm in adding another test using `Guid` array, will add it.

Reply:
Added another unit test for Guid array duplicate field.

Reply:
Thank you, Babu! 🙂